### PR TITLE
PT-2170: Move countries list getting into a service

### PR DESCRIPTION
**Pull request by eMazeika**
https://virtocommerce.atlassian.net/browse/PT-2170

### CODE REVIEW

**Comment by mvktsk** [comment]
Review task https://virtocommerce.atlassian.net/browse/VP-7475 has been created

File: src/VirtoCommerce.Platform.Web/VirtoCommerce.Platform.Web.csproj [hunk 1]
```diff
@@ -8,6 +8,7 @@
         <DockerComposeProjectPath>..\..\docker-compose.dcproj</DockerComposeProjectPath>
         <DockerDefaultTargetOS>Linux</DockerDefaultTargetOS>
         <DockerfileContext>..\..</DockerfileContext>
+        <UserSecretsId>7e57cba1-b67b-45a1-86c1-749174e42fc4</UserSecretsId>
```
Comment (Vectorfield4) [inline src/VirtoCommerce.Platform.Web/VirtoCommerce.Platform.Web.csproj:11]: It looks like private data.

File: src/VirtoCommerce.Platform.Data/Common/FileSystemCountriesService.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        {
+            var cacheKey = CacheKey.With(GetType(), nameof(GetCountriesAsync));
+            var results = await _memoryCache.GetOrCreateExclusiveAsync(cacheKey, async (cacheEntry) =>
+            {
+                var filePath = Path.GetFullPath("app_data/countries.json");
+                return JsonConvert.DeserializeObject<IList<Country>>(await File.ReadAllTextAsync(filePath));
+            });
+
+            return results;
+        }
```
Comment (Vectorfield4) [inline src/VirtoCommerce.Platform.Data/Common/FileSystemCountriesService.cs:33]:
```suggestion
        public Task<IList<Country>> GetCountriesAsync()
        {
            var cacheKey = CacheKey.With(GetType(), nameof(GetCountriesAsync));
            return _memoryCache.GetOrCreateExclusiveAsync(cacheKey, async (cacheEntry) =>
            {
                var filePath = Path.GetFullPath("app_data/countries.json");
                return JsonConvert.DeserializeObject<IList<Country>>(await File.ReadAllTextAsync(filePath));
            });
        }
```

**Review by Vectorfield4 — CHANGES_REQUESTED** [review]

**Comment by Vectorfield4** [comment]
Add key (PT-****) to the title, please.

**Review by akak1977 — APPROVED** [review]

**Review by Vectorfield4 — APPROVED** [review]

**Pull request merged**